test/bin/scenario.sh

```diff
@@ -137,19 +143,21 @@ sos_report_for_vm() {
     # version.
     cat - >/tmp/sos-wrapper.sh <<EOF
 #!/usr/bin/env bash
-if [ -f /usr/bin/microshift-sos-report ]; then
-    sudo /usr/bin/microshift-sos-report
+if ! hash sos ; then
+    echo "ERROR: The sos command does not exist"
+elif [ -f /usr/bin/microshift-sos-report ]; then
+    /usr/bin/microshift-sos-report || echo "ERROR: The /usr/bin/microshift-sos-report script failed"
 else
-    sudo chmod +x /tmp/microshift-sos-report.sh
-    sudo PROFILES=network,security /tmp/microshift-sos-report.sh
+    chmod +x /tmp/microshift-sos-report.sh
+    PROFILES=network,security /tmp/microshift-sos-report.sh || echo "ERROR: The /tmp/microshift-sos-report.sh script failed"
 fi
-sudo chmod +r /tmp/sosreport*
+chmod +r /tmp/sosreport-* || echo "ERROR: The sos report files do not exist in /tmp"
```

Comment:
Why is the sudo command removed from this line? I think we need it to change the permissions on files owned by root, don't we?

Reply:
Because the script is called with sudo already and it all runs elevated. So, all sudo in the script were removed for better readability.